neurow/integration_test/message_brokering_test.exs

```diff
@@ -34,7 +34,7 @@ defmodule Neurow.IntegrationTest.MessageBrokeringTest do
 
           assert_headers(headers, [
             {"access-control-allow-origin", "*"},
-            {"cache-control", "no-cache"},
+            {"cache-control", "no-cache, no-store"},
```

Comment:
shall we do exactly as siiloSSE `no-cache, no-store, max-age=0, must-revalidate`?